src/utils/dispatcher.h

```diff
@@ -0,0 +1,54 @@
+#ifndef DISPATCHER_H
+#define DISPATCHER_H
+#include <memory>
+#include <glibmm.h>
+
+namespace utils {
+    /**
+     * @brief The Dispatcher, executes given function on gtk main loop.
```

Comment:
This looks like a doxygen tag but it's the only one I can see. Does gTox use doxygen? If not, what is this?